python/ray/serve/_private/http_util.py

```diff
@@ -64,7 +64,10 @@ def convert_object_to_asgi_messages(
         body = obj.encode("utf-8")
         content_type = b"text/plain; charset=utf-8"
     else:
-        body = json.dumps(jsonable_encoder(obj, custom_encoder=serve_encoders)).encode()
+        body = json.dumps(
+            jsonable_encoder(obj, custom_encoder=serve_encoders),
+            separators=(",", ":"),
```

Comment:
do you mind providing a comment why we need to have separators? (Very difficult to understand why we need it)